presto-native-execution/presto_cpp/main/common/Utils.h

```diff
@@ -34,4 +34,12 @@ std::shared_ptr<folly::SSLContext> createSSLContext(
 /// Returns current process-wide CPU time in nanoseconds.
 long getProcessCpuTimeNs();
 
+/// Install a custom signal handler.
+/// On MacOS use a Google based implementation and on
+/// Linux (other platforms) use a Folly (Velox) based implementation.
```

Comment:
@czentgr 
nit: But can you also add here that the Folly based one provides more information about queryId etc.. as you described in the commit? Thanks.